pkg/kubelet/prober/common_test.go

```diff
@@ -108,6 +108,7 @@ func newTestManager() *manager {
 	// Add test pod to pod manager, so that status manager can get the pod from pod manager if needed.
 	podManager.AddPod(getTestPod())
 	m := NewManager(
+		podManager,
```

Comment:
Yeah, the configsource always sends a copy of the pod to config source channel,so we added podManager here.